Dockerfile.local

```diff
@@ -1,7 +1,7 @@
 FROM node:gallium-alpine
 WORKDIR /app
 COPY package.json package-lock.json ./
-RUN npm i --force
+RUN npm i --legacy-peer-deps
```

Comment:
legacy-peer-deps zal geen peer dependencies installeren. Of doet dat toch op mijn local machine niet.
Maar als het build op jenkins is het voor mij goed.